README.md

```diff
@@ -182,6 +182,8 @@ You can access a clickable demo with many examples and a [live code editor](http
 | **slides** | *Array* | `undefined` | Alternative way to pass slides. This prop expects an array of JSX <img> elements |
 | [**slidesPerPage**](https://brainhubeu.github.io/react-carousel/docs/examples/multipleItems) | *Number* | `1` | Number of slides visible at once |
 | [**slidesPerScroll**](https://brainhubeu.github.io/react-carousel/docs/examples/slidesPerScroll) | *Number* | `1` | Number by which value will change on scroll (autoPlay, arrow click, drag)|
+| [**lazyLoad**](https://brainhubeu.github.io/react-carousel/docs/examples/lazyload) | *Boolean* | `false` | Initially, it loads only the closest next / previous slide to the current one based on the value of the slidesPerPage variable. Other slides are loaded as needed. |
```

Comment:
according to this spec lazyload by default should disabled but in the example, you provided lazyload works despite that option not being provided. Is it intended behavior?

Reply:
perhaps it's a good idea to update gatsby docs with lazyload prop description. WDY?

Reply:
> according to this spec lazyload by default should disabled but in the example, you provided lazyload works despite that option not being provided. Is it intended behavior?
thanks, looks like I forgot to set it to false in defaultProps.
> perhaps it's a good idea to update gatsby docs with lazyload prop description. WDY?
It is there from the first commit: https://beghp.github.io/gh-pages-rc-10/docs/examples/lazyload . Oh, you mean API section in docs? I've didn't noticed that before

Reply:
ok, thanks for the answer! The PR LGTM